src/ds5/ds5-private.h

```diff
@@ -649,6 +708,32 @@ namespace librealsense
             motion_module_fw_version_offset = 212
         };
 
+        enum gvd_sc_fields      // gvd fields for Safety Camera
```

Comment:
I know it wasn't before, but maybe use enum class?
When the enum is define in a header file enum class will protect unintended assignments